src/modeling/models/mmt_encoder.py

```diff
@@ -118,7 +121,7 @@ def __init__(self,
      self._embedding_dropout_layer = tf.keras.layers.Dropout(
          rate=hidden_dropout_prob, name='embeddings/dropout')
 
-     self._relative_transformer = etc_layers.RelativeTransformerLayers(
+     self._transformer_layers = etc_layers.RelativeTransformerLayers(
```

Comment:
Why do we need relative transformer?

Reply:
We use relative transformer as our backbone.